build_projects/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -175,14 +175,14 @@ public static BuildTargetResult DownloadHostAndSharedFxArchives(BuildTargetConte
 
             Mkdirp(Path.GetDirectoryName(combinedSharedHostAndFrameworkArchiveDownloadFile));
 
-            if ( ! File.Exists(combinedSharedHostAndFrameworkArchiveDownloadFile))
+            if (!File.Exists(combinedSharedHostAndFrameworkArchiveDownloadFile))
```

Comment:
So many times I've skimmed over the `!` I just prefer this so it stands out. 

---
In reply to: [67968817](https://github.com/dotnet/cli/pull/3669#discussion_r67968817) [](ancestors = 67968817)